raiden-ts/src/transfers/epics/expire.ts

```diff
@@ -59,12 +64,12 @@ export function transferAutoExpireEpic(
               action$,
               transferExpire.request(undefined, meta),
               isResponseOf(transferExpire, meta),
-            ),
+            ).pipe(delay(confirmationBlocks * blockTime)),
```

Comment:
The idea here is just to give some time to partner to also see that the transfer is ready to be expired. In case there's a race/clockskell and the partner doesn't accept this message, it'll get retried and eventually accepted, but we try _a little_ to avoid the race by shifting it in time here. It is more relevant on block-based code, but shouldn't cause problems on timestamp or confirmationBlocks=0 either.

Reply:
Okay, that makes sense. Would be amazing if the code could express this.